### PR TITLE
Fix: Added Responsiveness to Dots Background

### PR DESCRIPTION
**Pull request by myselfprincee**
Added Responsiveness to the dots Background. Please Review the change @eduardconstantin  and let me know what I can do better :)

### CODE REVIEW

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

File: components/DotsBackground.js [hunk 1]
```diff
@@ -1,17 +1,47 @@
+'use client';
+
 import React from 'react';
 import styles from './DotsBackground.module.css';
 
 const DotsBackground = () => {
-  const numCols = 100;
+  const numCols =
```
[the remark this answers is not in the document]
Comment (myselfprincee) [reply components/DotsBackground.js:7]: I have added that if the width is larger than 1000px then, the columns will be 35.

[the remark this answers is not in the document]
Comment (myselfprincee) [reply components/DotsBackground.js:7]: on the else condition of the function

[the remark this answers is not in the document]
Comment (eduardconstantin) [reply components/DotsBackground.js:7]: i see, but if the with is 4000px, 35 columns won't be enough

[the remark this answers is not in the document]
Comment (myselfprincee) [reply components/DotsBackground.js:7]: : window.innerWidth <= 800
      ? 20
      : window.innerWidth <= 850
      ? 21
      : window.innerWidth <= 900
      ? 22
      : window.innerWidth <= 950
      ? 23
      : window.innerWidth <= 1000
      ? 24
      **: 35**;

  const rowSpacing = window.innerWidth > 1000 ? 3 : **2**;

The thing is that the js has to go through 8-10 steps to check for the cols to set. I couldn't think of any other solution as I am myself a beginner i'd say.

[the remark this answers is not in the document]
Comment (eduardconstantin) [reply components/DotsBackground.js:7]: no, it's ok, i'll merge it like this

**Pull request merged**